features/step_definitions/info_steps.rb

```diff
@@ -0,0 +1,3 @@
+Given(/^that I visited on the (\d+)th March (\d+)$/) do |arg1, arg2|
```

Comment:
Unused block argument - `arg1`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `arg2`. You can omit all the arguments if you don't care about them.